db/seeds.rb

```diff
@@ -1,7 +1,232 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+admin_user1 = User.find_by(email: "historybuff40@example.com")
+unless admin_user1.present?
+  admin_user1 = User.create(
+    user_name: "history_buff",
+    email: "historybuff40@example.com",
+    password: "inthepast",
```

Comment:
You might want to factor this out into an environment variable since now you're basically publishing the password of a user with full permissions to destroy your site on Github, haha.